website/members.md

```diff
@@ -0,0 +1,73 @@
+EMP membership is defined by participation {style="text-align: left"}
+------------------------------------------
+
+**Project Management and Steering Committee** comprises:
+
+**Jack A. Gilbert**, **Folker Meyer** and **Rick Stevens** from Argonne
+National Laboratory and University of Chicago, **Jonathan Eisen**
+(University of California, Davis), **Jed Fuhrman** (University of
+Southern California), **Janet Jansson** (Lawrence Berkley National
+Laboratory), **Rob Knight** and **Noah Fierer** (University of Colorado,
+Boulder), **Mark Bailey** (Center for Ecology and Hydrology, UK),
+**George Kowalchuk** (Netherlands Institute of Ecology), .
```

Comment:
I agree. Here is the link to send around.
https://github.com/colinbrislawn/emp/blob/patch-1/website/members.md